styles.css

```diff
@@ -159,3 +156,409 @@ header {
   height: 1.25rem;
   width: 1.25rem;
 }
+
+.My-Recent {
+  display: flex;
+  flex-direction: column;
+  padding: 1.5rem;
+  flex-wrap: nowrap;
+}
```

Comment:
- [ ] Kindly consider using `display grid` here in the works container instead of display flex as per the requirements.
   - In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.